src/Raven.Client/Exceptions/ConcurrencyException.cs

```diff
@@ -40,21 +41,51 @@ public ConcurrencyException(string message, Exception inner) : base(message, inn
         /// <summary>
         /// Expected Etag.
         /// </summary>
-        public long ExpectedETag { get; set; }
+        public long ExpectedETag;
```

Comment:
this is a breaking change. Let's stay with properties in this exception

Reply:
mark this maybe as obsolete?

Reply:
done